### DISCUSSION TITLE
Need help requesting different profiles of an account

### DISCUSSION THREAD

**Discussion by igarshasb** [LeleDerGrasshalmi/FortniteEndpointsDocumentation#351] [Q&A] [answered]
Hello. 
I need help with getting different profiles for an account. Specifically regarding [THIS](https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/4c2d560367107eed6c8e4b1ae069f646b136f61b/EpicGames/FN-Service/Game/Profile/README.md?plain=1#L26)
Please guide me on how to authenticate with the account and request these profiles. Sorry for my beginner question :( I have encountered a bot that has provided me with the campaign profile(in .JSON) after authentication, I need to request the other profiles as well. Also I would appreciate if anyone could guide me in general on how to understand the syntax so I could also request files such as world/info. Many thanks in advance.

**Comment by LeleDerGrasshalmi** [comment, marked as answer]
simple auth guide: https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/EpicGames/AccountService/Authentication/Token.md

general profile operation docs: <https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/EpicGames/FN-Service/Game/Profile/README.md>

query profile (read profile): https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/EpicGames/FN-Service/Game/Profile/Operations/QueryProfile.md

**Reply by igarshasb** [reply]
Thanks for the reply. I have managed to do the authentication part. Could you please tell me exactly how I should request to download those profiles using httpie? The auth should be the access token I get from the authentication process.(right?) What should I put instead of :operation, and as parameters, body and header? Here's how it looks like right now with the "account id" redacted. I would sincerely appreciate it if you could guide me through this. Let's say I want to download outpost0 or campaign.
![Screenshot 2024-11-16 163343](https://github.com/user-attachments/assets/9815daa4-fba6-4779-bd1e-9b859b5701e3)

**Reply by LeleDerGrasshalmi** [reply]
You should use the headers/url from the docs and then use the body/operation name from the individual operation docs

`https://fngw-mcp-gc-livefn.ol.epicgames.com/fortnite/api/game/v2/profile/:accountId/client/QueryProfile`

**Reply by igarshasb** [reply]
OMG I had somehow deleted the /profile from the url by mistake and the errors were all a result of that. Sorry to have taken your time for this. If I enconuter any other erros I will update this.

**Reply by LeleDerGrasshalmi** [reply]
ok great, thanks for tge update.

**Reply by igarshasb** [reply]
> ok great, thanks for tge update.

Thank you!I still need help to download specific profiles though. Please guide me through this last step. I can now get the common_core profile. How do I change it to get campaign?

**Reply by LeleDerGrasshalmi** [reply]
<https://github.com/LeleDerGrasshalmi/FortniteEndpointsDocumentation/blob/main/EpicGames/FN-Service/Game/Profile/README.md>

![image](https://github.com/user-attachments/assets/d9dcc6af-0bb0-4fda-a9f1-9ee87a43b698)

**Reply by igarshasb** [reply]
I can't thank you enough. Figured it out right as you replied. Thanks again for the time you put in to help me. Really appreciate it.
